### DISCUSSION TITLE
Central Scene values are not set as "stateless"

### DISCUSSION THREAD

**Discussion by kpine** [zwave-js/zwave-js#6455] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

Testing out some new HA functionality to represent value notifications as event entities. These support Central Scene devices. An entity should be created for each key attribute (at least I think that's how it works). These entities are not being created. The [discovery schema](https://github.com/home-assistant/core/blob/6372bc3aaa7af00d4ca3d1064d59978ea1c1ce60/homeassistant/components/zwave_js/discovery.py#L1050-L1058) for these relies on the event values to have a `stateful: false` value. I am seeing the CS values has having stateful values, despite #6424. These is even after re-interviewing the device.

```json
        "39-91-0-scene-001": {
          "endpoint": 0,
          "commandClass": 91,
          "commandClassName": "Central Scene",
          "property": "scene",
          "propertyKey": "001",
          "propertyName": "scene",
          "propertyKeyName": "001",
          "ccVersion": 3,
          "metadata": {
            "type": "number",
            "readable": true,
            "writeable": false,
            "label": "Scene 001",
            "min": 0,
            "max": 255,
            "states": {
              "0": "KeyPressed",
              "1": "KeyReleased",
              "2": "KeyHeldDown",
              "3": "KeyPressed2x",
              "4": "KeyPressed3x",
              "5": "KeyPressed4x",
              "6": "KeyPressed5x"
            },
            "stateful": true,
            "secret": false
          }
        },
```

What did you expect to happen instead?

The CS values should show `"stateful": false`.

  1. Have a device that supports Central Scene
  2. Examine the value information, either with HA device diagnostic or ZUI debug
  3. See CS values are set to `"stateful": true,`, but should be false.

Anything else we should know?

### Software versions

zwave-js-ui: 9.2.3.1454eca
zwave-js: 12.2.1

### Device information

Manufacturer: HomeSeer
Model name: HS-FC200+
Node ID: 39

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave_js.log](https://github.com/zwave-js/node-zwave-js/files/13096916/zwave_js.log)

[zwave_js-3aea29d0aede8f900a111601a3df8501-Scene Capable Fan Control Switch-8b51f41930b70c0513fccf92d223bb4f.json.txt](https://github.com/zwave-js/node-zwave-js/files/13096995/zwave_js-3aea29d0aede8f900a111601a3df8501-Scene.Capable.Fan.Control.Switch-8b51f41930b70c0513fccf92d223bb4f.json.txt)

[node_debug.json.txt](https://github.com/zwave-js/node-zwave-js/files/13097421/node_debug.json.txt)

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
Fixed by #6457 and #6458
